content/docs/react-without-jsx.md

```diff
@@ -1,14 +1,14 @@
 ---
 id: react-without-jsx
-title: React Without JSX
+title: JSX के बिना React
 permalink: docs/react-without-jsx.html
 ---
 
-JSX is not a requirement for using React. Using React without JSX is especially convenient when you don't want to set up compilation in your build environment.
+React का उपयोग करने के लिए JSX की आवश्यकता नहीं है। JSX के बिना React का उपयोग करना विशेष रूप से तब सुविधाजनक होता है जब आप अपने बील्ड परिवेश मे कंपाइलेशन को स्थापित नहीं करना चाहते हैं।
 
-Each JSX element is just syntactic sugar for calling `React.createElement(component, props, ...children)`. So, anything you can do with JSX can also be done with just plain JavaScript.
+प्रत्येक JSX एलिमेंट `React.createElement(component, props, ...children)` को कॉल करने के लिए मात्र सुंदर लिखने का तरीक़ा है। अतः, आप JSX से जो कुछ भी कर सकते हैं, वही सब सामान्य जावास्क्रिप्ट से भी किया जा सकता है।
 
-For example, this code written with JSX:
+उदाहरण के लिए, यह कोड JSX के साथ लिखा गया है:
```

Comment:
`यह कोड JSX के साथ लिखा गया है:` 

This should be

`ye JSX ke sath likha gya code:`

Reply:
how about `यह JSX के साथ लिखा गया कोड है:` ?

Reply:
sounds good